client/packages/programs/src/JsonForms/useJsonForms.tsx

```diff
@@ -129,6 +132,12 @@ export const useJsonForms = <R,>(
 
   useConfirmOnLeaving(isDirty);
 
+  const updateSubmitActions = useCallback((key: string, action: () => void) => {
```

Comment:
Wondering if `registerSubmitAction` would be clearer?

Reply:
Yup, was thinking something along those lines